apps/src/music/progress/MusicValidator.ts

```diff
@@ -23,6 +24,10 @@ export const MusicConditions: ConditionNames = {
     valueType: 'number',
   },
   PLAYED_SOUND_TRIGGERED: {name: 'played_sound_triggered'},
+  PLAYED_SOUND_TRIGGERED_MULTIPLE_TIMES: {
```

Comment:
Making sure I understand - what exactly is this condition checking for? Is this checking if a single trigger was pressed multiple times?

Reply:
Correct! I’ll be adding a description value for levelbuilders once the other branch merges. I’ve set myself up perfectly for a merge conflict too. :D

Reply:
Gotcha, thanks! I have a question about unique function ID but i'll add that to the relevant line.